File: typescript/docs_code_snippets/functions-delete.ts
import { z } from "zod";
import { Opper } from "opperai";

const opper = new Opper();

// setup
const fnName = `docs-snippet-fn-${Date.now()}`;
await opper.call(fnName, { input: "hello", output_schema: z.object({ reply: z.string() }) });
// /setup

// --- docs ---
await opper.functions.delete(fnName);
// --- /docs ---

console.log("Function deleted");

File: python/docs_code_snippets/functions-delete.py
import time
from opperai import Opper

opper = Opper()

# Setup: create function via call
fn_name = f"docs-snippet-{int(time.time())}"
opper.call(fn_name, input="hello")

# --- docs ---
opper.functions.delete(fn_name)
# --- /docs ---
